packages/utilities/gel-foundations/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/gel-foundations",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

```suggestion
  "version": "6.1.0",
```
I think it may be better to bump this a minor version as you are adding functionality [semver](https://semver.org/) says:
> MINOR version when you add functionality in a backwards compatible manner

You have added functionality but you haven't broken any existing functionality (a non-breaking change). Hope that makes sense?